startup.d/10-git-pull.sh

```diff
@@ -1,2 +1,21 @@
 #!/bin/sh
+
+# Prepare code auditing environment
+REPO_URL=$(git remote get-url origin)
+TEST_DIR=$(mktemp -d)
+git clone "$REPO_URL" "$TEST_DIR";
+pushd "$TEST_DIR";
+
+# Code auditing section
+python -m py_compile $(find . -name '*.py');
```

Comment:
@ad-m @reddraggone9 yep the chaos venv python lives in `/root/.virtualenvs/chaos/bin/python`.  We should do something so we don't have to hardcode it for too much longer in our script files